platform/platform-configuration-migration/src/main/java/org/codice/ddf/configuration/migration/SystemConfigurationMigration.java

```diff
@@ -217,7 +231,12 @@ private void copyDirectory(Path source, Path destination) throws MigrationExcept
             String message = String.format(UNREAL_PATH_WARNING, propertyName, path.toString());
             LOGGER.debug(message);
             migrationWarnings.add(new MigrationWarning(message));
+        } catch (UnsupportedOperationException e) {
```

Comment:
Since we're not trying to create a symlink but are just checking to see if a path contains a symbolic link, should we ever get this exception, even on Windows?